website/src/dynamic/users.yml

```diff
@@ -458,3 +458,8 @@
   image: "img/companies/zozo-technologies.png"
   infoLink: "https://tech.zozo.com/en/"
   pinned: false
+-
+  caption: Giant Monkey GmbH
+  image: "img/companies/gm.png"
+  infoLink: "https://www.giantmonkey.de/"
+  pinned: true
```

Comment:
What about following the alphabetical order to put it somewhere in the middle of the list?

Reply:
Done!